### ISSUE TITLE
Error with VALUES and AT TIME ZONE

### ISSUE THREAD

**Issue by jdmaturen** [prestodb/presto#6420]
```
presto> select t from (values (from_unixtime(8) at time zone 'UTC')) as x (t);
Query 20161022_212404_00013_z7ztv failed: not yet implemented: com.facebook.presto.sql.tree.AtTimeZone

presto> select t at time zone 'UTC' from (values (from_unixtime(8))) as x (t);
            _col0
-----------------------------
 1970-01-01 00:00:08.000 UTC
(1 row)

Query 20161022_212416_00014_z7ztv, FINISHED, 1 node
Splits: 1 total, 1 done (100.00%)
0:00 [0 rows, 0B] [0 rows/s, 0B/s]

presto>
```

Expectation is that both queries would have the same result.

Tested in master as of 76137c5aff5d5704ea9999e62d6d9d2fa302bacc.

**Comment by geraint0923** [comment]
Should be fixed by #6423

**Comment by jdmaturen** [comment]
@geraint0923 thanks!
